day8/alienalphabet.py

```diff
@@ -0,0 +1,127 @@
+##############
+## Solution ##
+##############
+
+def alienAlphabet(orderedWords):
+    # edge cases
+    if len(orderedWords) <= 0:
+        return ''
+    if len(orderedWords) == 1:
+        return removeDups(orderedWords[0])
```

Comment:
What does this do? If your dictionary only has one word and more than one letter, you can't determine anything.